ios/AppLovinMAX.m

```diff
@@ -1359,6 +1388,23 @@ - (void)positionAdViewForAdUnitIdentifier:(NSString *)adUnitIdentifier adFormat:
     [NSLayoutConstraint activateConstraints: constraints];
 }
 
+- (void)setPendingExtraParametersIfNeeded:(ALSdkSettings *)settings
+{
+    NSDictionary *extraParameters;
+    @synchronized ( _extraParametersToSetLock )
+    {
+        if ( _extraParametersToSet.count <= 0 ) return;
+            
+        extraParameters = [NSDictionary dictionaryWithDictionary: _extraParametersToSet];
+        [_extraParametersToSet removeAllObjects];
+    }
+        
+    for ( NSString *key in extraParameters.allKeys )
+    {
+        [settings setExtraParameterForKey: key value: extraParameters[key]];
+    }
+}
```

Comment:
Don't all bridge calls into React Native occur on the main thread? (such as when the pub sets extra parameter or inits the plugin). If so, we won't need the lock.

Reply:
Yes, they occur not on the main thread.    Removed the lock, and updated and tested.